recipes/sentry-native/all/conanfile.py

```diff
@@ -77,16 +78,18 @@ def package(self):
         cmake = self._configure_cmake()
         cmake.install()
         tools.rmdir(os.path.join(self.package_folder, "lib", "cmake"))
+        for pdb in glob.glob(os.path.join(self.package_folder, "bin", "*.pdb")):
+            os.unlink(pdb)
 
     def package_info(self):
         self.cpp_info.libs = ["sentry"]
-        if self.settings.os in ("Android", "Windows"):
-            self.cpp_info.exelinkflags= ["--build-id=sha1"]
-            self.cpp_info.sharedlinkflags = ["--build-id=sha1"]
+        if self.settings.os in ("Android", "Linux"):
```

Comment:
no longer needed for Windows?

Reply:
I think it was a mistake. The build id is only defined for Android and Linux in the CMakeLists(See https://github.com/getsentry/sentry-native/blob/7fec6556143eb6332e5306537ef55be2e4923d1b/CMakeLists.txt#L279)